translations/README.korean.md

```diff
@@ -0,0 +1,221 @@
+
+
+### 리얼타임_어플리케이션
+
+- React 기반으로 만든 그룹챗 어플리케이션, typing indicator를 포함한, 온라인 유저 & 메세지 수신 알람
+  - [Try it out](https://realtime-chat.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-chat)
+  - [Browse APIs](https://realtime-chat.demo.hasura.app/console)
+
+- 달리는 차량 에서 현재 GPS 위치를 지도에 실시간으로 보여주는 위치 트래킹 앱
+  - [Try it out](https://realtime-location-tracking.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-location-tracking)
+  - [Browse APIs](https://realtime-location-tracking.demo.hasura.app/console)
+
+- 지속적으로 변화되는 데이터를 집계하는 리얼타임 대시보드
+  - [Try it out](https://realtime-poll.demo.hasura.app/)
+  - [Tutorial](community/sample-apps/realtime-poll)
+  - [Browse APIs](https://realtime-poll.demo.hasura.app/console)
+
+### 영상
+
+* [개인 호스트 gitlab 인스턴스에 GraphQL 추가하기](https://www.youtube.com/watch?v=a2AhxKqd82Q) (*3:44 분*)
+* [Auth0와 GraphQL 백엔드가 있는 Todo app](https://www.youtube.com/watch?v=15ITBYnccgc) (*4:00 분*)
+* [GitLab 인증과 통합된 GitLab의 GraphQL](https://www.youtube.com/watch?v=m1ChRhRLq7o) (*4:05 분*)
+* [천만대의 탈것와 위치기반(PostGIS, Timescale) 을 위한 대시보드](https://www.youtube.com/watch?v=tsY573yyGWA) (*3:06 분*)
+
+
+## 지원_&_트러뷸슈팅
+
+문서와 커뮤니티는 대부분의 트러뷸 슈팅을 해결하는데 도움을 줄겁니다. 만약 버그를 발견했거나 저희의 도움이 필요하다면 아래 첨부된 채널중 하나로 연락 해주세요
+
+* 지원 & 피드백: [Discord](https://discord.gg/vBPpJkS)
+* 이슈 & 버그 트래커: [GitHub issues](https://github.com/hasura/graphql-engine/issues)
+* 프로덕트 업데이트: [@HasuraHQ](https://twitter.com/hasurahq)
+* Talk to us on our [website chat](https://hasura.io)
+* 저희 [웹사이트](https://hasura.io) 에서 대화할수있어요
+
+저희는 개방적이고 환영받는 커뮤니티 환경을 조성하는데 최선을 다하고 있습니다. 더 많은 커뮤니티 가이드를 보고 싶으시다면 [행동 강령](code-of-conduct.md) 를 참고해주세요.
+
+만약 보안이슈에 대해 리포트를 하고싶으시다면, [이것](SECURITY.md)을 봐주세요.
+
+## 기여
+
+자세한 정보는 [컨트리뷰트 가이드](CONTRIBUTING.md) 에서 확인하실수 있습니다.
+
+## 브랜드_자산
+
+Hasura 브랜드 자산 (로고, 하수라 마스코트, 뱃지 등등.) 은 [애셋/브랜드](../assets/brand) 폴더에서 찾을수있어요.
+이것들을 당신의 어플리케이션, 웹사이트에 마음껏 쓰세요. 
+만약 Hasura로 만든 어플리케이션에 "Powered by Hasura" 배지를 넣어주신다면 저희는 더 감격스러울것 같아요 ❤️
+
+<div style="display: flex;">
+  <img src="../assets/brand/powered_by_hasura_blue.svg" width="150px"/>
+  <img src="../assets/brand/powered_by_hasura_white.svg" width="150px"/>
+</div>
+
+```html
+<!-- For light backgrounds -->
+<a href="https://hasura.io">
+  <img width="150px" src="https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_blue.svg" />
+</a>
+
+<!-- For dark backgrounds -->
+<a href="https://hasura.io">
+  <img width="150px" src="https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_white.svg" />
+</a>
+```
+
+## 라이선스
+
+GraphQL Engine core 는 [아파치 라이선스 2.0](https://www.apache.org/licenses/LICENSE-2.0) (Apache-2.0)로 적용이됩니다.
+
+모든 **다른 컨텐츠들은** ([`server`](server), [`cli`](cli) 그리고 [`console`](console) 디렉토리를 제외한) 모든것들은 [MIT 라이선스](LICENSE-community)로 적용이 됩니다.
+이건 [`docs`](docs) 와 [`community`](community) 내에 있는 모든것들도 포함이 됩니다.
+
+## 번역
+
+이 readme 는 다른 번역으로 제공됩니다. :
+
+- [Korea :kr:](translations/README.korean.md) (:pray: [@라스크](https://github.com/laskdjlaskdj12))
```

Comment:
Could you put the Korean translations to the end, after Chinese, please? So that we have it in the order that they were created 🙂